app/views/staff/proposals/show.html.haml

```diff
@@ -43,7 +43,6 @@
       = render partial: 'proposals/confirmation_notes', locals: {proposal: proposal, notes: proposal.confirmation_notes}
 
     .col-md-4
```

Comment:
Its actually for the Review / Ratings section as well. Also for internal comments.